ArticleTemplates/liveblogTemplateContributionsEpic.html

```diff
@@ -0,0 +1,8 @@
+<div class="contributions-epic" id="__CONTRIBUTIONS_EPIC_ID__">
+    <style class="contributions-epic__styling">
+        // Contributions Epic styling will go here
```

Comment:
The comments will need to be HTML comments as they'll be within the flow of the HTML